### PR TITLE
[Reloaded] [C-1017] Fix more by artist lineup

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
Fix the more by artist lineup not loading. The issue was that the request needed the track permalink and artist handle which mobile wasn't passing in.
Also fixes a subsequent issue where the lineup wasn't resetting.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Review by dylanjeffers — APPROVED** [review]
Awesome find, that's a tough one

File: packages/web/src/common/store/pages/track/sagas.js [hunk 1]
```diff
@@ -171,7 +170,11 @@ function* watchFetchTrack() {
         // Add hero track to lineup early so that we can play it ASAP
         // (instead of waiting for the entire lineup to load)
         yield call(addTrackToLineup, track)
-        yield fork(getRestOfLineup, permalink, handle)
+        yield fork(
+          getRestOfLineup,
+          track.permalink,
```
Comment (nicoback2) [inline packages/web/src/common/store/pages/track/sagas.js:175]: All tracks have a `permalink` field right? cc @raymondjacobson

Comment (sliptype) [reply packages/web/src/common/store/pages/track/sagas.js:175]: @rickyrombo might have more context here too

File: packages/web/src/common/store/pages/track/sagas.js [hunk 2]
```diff
@@ -136,7 +136,6 @@ function* getRestOfLineup(permalink, ownerHandle) {
 function* watchFetchTrack() {
   yield takeEvery(trackPageActions.FETCH_TRACK, function* (action) {
     const { trackId, handle, slug, canBeUnlisted } = action
-    const permalink = `/${handle}/${slug}`
```
Comment (sliptype) [inline packages/web/src/common/store/pages/track/sagas.js:139]: Do some tracks not have handle or slug? Why the change here

**Review by sliptype — APPROVED** [review]

File: packages/web/src/common/store/pages/track/sagas.js (hunk 1 above)
Comment (nicoback2) [reply packages/web/src/common/store/pages/track/sagas.js:175]: @rickyrombo do u kno ^ :>)?

File: packages/web/src/common/store/pages/track/sagas.js (hunk 2 above)
Comment (nicoback2) [reply packages/web/src/common/store/pages/track/sagas.js:139]: oh i just got rid of it because it was no longer needed after the `track.permalink` change below. `Handle` and `slug` are optional params for this action and are not passed in by mobile (since it only gets the `trackId` from the router

File: packages/web/src/common/store/pages/track/sagas.js (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/common/store/pages/track/sagas.js:175]: Yes they should!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--fix-tracks-liineup
